UnoCheck/Program.cs

```diff
@@ -51,6 +51,12 @@ static Task<int> Main(string[] args)
 
 			app.Configure(config =>
 			{
+                var version = ToolInfo.CurrentVersion;
+                var buildDate = File.GetLastWriteTime(typeof(ToolInfo).Assembly.Location)
+                    .ToString("yyyy-MM-dd");
```

Comment:
won't this be the install (not build) date ?

I have the same `uno-check` on two Macs but their dates (on the file system) are different